ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -315,8 +318,12 @@ class DrawerPresentationController: UIPresentationController {
         var contentFrame = bounds.inset(by: marginsForContentView())
 
         var contentSize = presentedViewController.preferredContentSize
+        let landscapeMode = contentFrame.width > contentFrame.height
```

Comment:
is this a good assumption? Can't the drawer have variable sizes, such that will make it longer or wider regardless of the device window state? Should we based this on the screen size instead?

Reply:
ah yes. thanks for pointing out that mistake. I will use the sourceViewcontroller's view.windows size.